predict.py

```diff
@@ -67,7 +67,11 @@ def create_network(network_input, n_vocab):
     model.compile(loss='categorical_crossentropy', optimizer='rmsprop')
 
     # Load the weights to each node
-    model.load_weights('weights.hdf5')
+    try:
+        model.load_weights(sys.argv[2])
+        print('Loaded weights from argument')
+    except NameError:
+        model.load_weights('weights.hdf5')
```

Comment:
Make it load new_weights.hdf5 instead of weights.hdf5 by default, I've been meaning to fix that but haven't gotten around to it so this would be the perfect place to throw that change in.